docs/Queries.md

```diff
@@ -63,6 +66,15 @@ GET requests) or the request body (for POST requests).
 An array of feature flag (Runtime Flag) names that should be returned
 alongside the main request.
 
+### `meta`
+
+#### `method`
+
+You can force the query to be sent with a particular HTTP method by specifying
+it here as a _lowercase_ string: `get`, `post`, `delete`, or `patch`. Note that
+you should never try to make a request that contains multiple queries with
+different `method`s.
```

Comment:
those would have to be synchronous anyway - you wouldn't want to send the GET before knowing whether the POST was successful. Teams could talk to API engineers to optionally return the created/updated object for a successful POST.